contexts/arbo-context/arbo-filter-update-steps.tsx

```diff
@@ -0,0 +1,29 @@
+import { MapRef } from "react-map-gl"
+import { ArboAction, ArboStateType } from "./arbo-context"
+import { pipe } from "fp-ts/lib/function.js";
+import { adjustMapPositionStep } from "./filter-update-steps/adjust-map-position-step";
+import { updatePediatricAgeGroupFilter } from "./filter-update-steps/update-pediatric-age-group-filter";
+import { addActionToSelectedFiltersStep } from "./filter-update-steps/add-action-to-selected-filters-step";
+import { applyNewSelectedFiltersStep } from "./filter-update-steps/apply-new-selected-filters-step";
+
+export interface HandleArboFilterUpdateInput {
+  state: ArboStateType,
+  action: ArboAction,
+  map: MapRef | undefined
+}
+
+export interface HandleArboFilterUpdateOutput {
+  state: ArboStateType,
+  action: ArboAction,
+  map: MapRef | undefined
+}
+
+export const handleArboFilterUpdate = (input: HandleArboFilterUpdateInput): HandleArboFilterUpdateOutput => {
+  return pipe(
+    input,
+    addActionToSelectedFiltersStep,
+    adjustMapPositionStep,
```

Comment:
nit: Only note would be to remove the step from the function names. I think using pipe indicates they are steps and the names are descriptive enough without the step. Could be good incase we want to do stuff outside the pipe (idk if needed)

Reply:
Yeah I think that's fair, I just realized I was being inconsistent with step/nostep anyhow so let's be consistent hahaha. Changed the name of all of the functions and file names for the individual steps. Kept the name of the folder as `filter-update-steps` and `arbo-filter-update-steps` but let me know if you think changing those makes sense too :)

ca80b92c183331b97ef5f68cacec17258c18c48d